File: CinemaRest.Service/CinemaRest.Service/Models/Character.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Web;

namespace CinemaRest.Service.Models
{
    public class Character : LinkResourceBase
    {
        public Guid Id { get; set; } = Guid.NewGuid();
        public string CharacterName { get; set; }
        public Actor Actor { get; set; }
        public Guid MovieId { get; set; }
        public bool Deleted { get; set; } = false;

        public Character() { }
    }
}

File: CinemaRest.Client/src/cinemarest/client/models/Character.java
package cinemarest.client.models;

public class Character {
    private String id;
    private String characterName;
    private String movieId;
    private boolean deleted;
    private Actor actor;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getCharacterName() {
        return characterName;
    }

    public void setCharacterName(String characterName) {
        this.characterName = characterName;
    }

    public String getMovieId() {
        return movieId;
    }

    public void setMovieId(String movieId) {
        this.movieId = movieId;
    }

    public boolean isDeleted() {
        return deleted;
    }

    public void setDeleted(boolean deleted) {
        this.deleted = deleted;
    }

    public Actor getActor() {
        return actor;
    }

    public void setActor(Actor actor) {
        this.actor = actor;
    }
}